src/test/ui/macros/issue-68058.rs

```diff
@@ -0,0 +1,15 @@
+// check-pass
+
+macro_rules! def_target {
+    ($target: expr) => {
+        #[target_feature(enable=$target)]
+        unsafe fn f() {
+            #[target_feature(enable=$target)]
+            ()
+        }
+    };
+}
+
+def_target!("avx2");
```

Comment:
@petrochenkov `// ignore-cloudabi no std::env` is enough to disable test on the unexpected environment?

Reply:
I don't think so.
Any target-neutral attribute (e.g. `#[doc = $expr]`) can be used instead, then these issues won't arise.

Reply:
Got it, fixed test case.